README.md

```diff
@@ -1 +1,28 @@
-# hw01
\ No newline at end of file
+
+---
+#"Computing for Social Sciences HW01"
+"**Taylor Griffin**"  
+"*October 8, 2019*"  
+
+# Introduction
+
+![Taylor Griffin](C:/Users/Taylor Griffin/Desktop/Computing R/Taylor G hw01/Taylor.png)
```

Comment:
Similar to above. One way to do this is to upload your image to your Github repository, click on it, and copy the URL from your URL bar. Then, in the markdown file type:

![Image Tag](URL you just copied)